scripts/classes/class.Extrausers.php

```diff
@@ -68,5 +77,20 @@ private static function _generateFile($file, $query, &$cronlog)
 		} else {
 			$cronlog->logAction(CRON_ACTION, LOG_NOTICE, 'Error when writing ' . $type . ' file entries');
 		}
+		if (chmod($file,$permissions) !== false) {
```

Comment:
I don't t hink you need to log when the permission-setting is successful (we almost never do).